qsv/test/org/qcmg/qsv/util/TestUtil.java

```diff
@@ -833,14 +840,18 @@ public static Clip getClip(String strand, String side, String name) {
 		return clips;
 	}
 
-	public static DiscordantPairCluster setupHiseqCluster(String clusterType, TemporaryFolder testFolder, String qPrimerCategory) throws IOException, Exception {
+	public static DiscordantPairCluster setupHiseqCluster(String clusterType, File testFolder, String qPrimerCategory) throws IOException, Exception {
 		  List<MatePair> pairs = new ArrayList<>();
 	      pairs.add(new MatePair("HWI-ST1240:47:D12NAACXX:7:1112:14008:49131:20120608113919562,chr10,89700049,89700149,AAC,97,false,HWI-ST1240:47:D12NAACXX:7:1112:14008:49131:20120608113919562,chr10,89712348,89712448,AAC,145,true,F1R2"));
 	      pairs.add(new MatePair("HWI-ST1240:47:D12NAACXX:4:1304:9295:38861:20120608092353631,chr10,89700053,89700153,AAC,97,false,HWI-ST1240:47:D12NAACXX:4:1304:9295:38861:20120608092353631,chr10,89712340,89712440,AAC,145,true,F1R2"));
 	      pairs.add(new MatePair("HWI-ST1240:47:D12NAACXX:7:1309:3779:55661:20120608113919562,chr10,89700060,89700160,AAC,97,false,HWI-ST1240:47:D12NAACXX:7:1309:3779:55661:20120608113919562,chr10,89712340,89712440,AAC,145,true,F1R2"));
 	      pairs.add(new MatePair("HWI-ST1240:47:D12NAACXX:6:2109:5161:48848:20120608110941621,chr10,89700064,89700164,AAC,97,false,HWI-ST1240:47:D12NAACXX:6:2109:5161:48848:20120608110941621,chr10,89712346,89712446,AAC,145,true,F1R2"));
 	      pairs.add(new MatePair("HWI-ST1240:47:D12NAACXX:8:1109:11792:69957:20120608020343585,chr10,89700200,89700300,AAC,161,false,HWI-ST1240:47:D12NAACXX:8:1109:11792:69957:20120608020343585,chr10,89712446,89712546,AAC,81,true,F2R1"));
-	      String tumourFile = testFolder.newFile("tumor.bam").getAbsolutePath();
+	     // String tumourFile = testFolder.newFile("tumor.bam").getAbsolutePath();
+	      File f = new File(testFolder,"tumor.bam" );
```

Comment:
why is this file being created?

Reply:
just follow the old method to create a file. But you are right, it is not necessary. I will delete this line.